yal/servicefinder_feedback_survey.py

```diff
@@ -179,10 +179,15 @@ async def state_servicefinder_feedback_confirmation(self):
     async def state_servicefinder_feedback_survey_2_start(self):
         # Repeat the question here for error and selection handling
         whatsapp_id = utils.normalise_phonenumber(self.inbound.from_addr).lstrip("+")
+        # Reset this, so that we only get the survey once after a push
+        await rapidpro.update_profile(whatsapp_id, {"feedback_survey_sent_2": ""})
         error, fields = await rapidpro.get_profile(whatsapp_id)
         if error:
             return await self.go_to_state("state_error")
-        feedback_datetime = datetime.fromisoformat(fields["feedback_timestamp"])
+        feedback_datetime = (
+            datetime.fromisoformat(fields["feedback_timestamp_2"])
+            - self.CALLBACK_2_DELAY
```

Comment:
I don't understand why we're subtracting this here now?

Reply:
This is an issue I picked up when doing this change. The `feedback_timestamp_2` is the timestamp of when the message goes out. So it will be today's date, which is not when they searched for a clinic. They searched for a clinic 2 weeks ago, and we're following up with them now.

Reply:
Ah I see. thanks for clarifying.